app/widgets/common/deploymentsView/header/RunWorkflowModal.tsx

```diff
@@ -29,39 +38,66 @@ interface RunWorkflowModalProps {
 
 const tModal = StageUtils.getT(`${i18nPrefix}.header.bulkActions.runWorkflow.modal`);
 
-const getWorkflowsOptions = (workflows: Workflow[]) => {
-    return _.chain(workflows)
+const getWorkflowOptionText = (workflow: EnhancedWorkflow) => {
+    return capitalize(workflow.name.replaceAll('_', ' '));
+};
+
+const getWorkflowOptions = (workflows: EnhancedWorkflow[]): DropdownItemProps[] => {
+    type OptionsGroupName = 'enabledOptions' | 'disabledOptions';
+    type GroupedOptions = Record<OptionsGroupName, DropdownItemProps[] | undefined>;
```

Comment:
Isn't it equal to:
```suggestion
    type GroupedOptions = Partial<Record<OptionsGroupName, DropdownItemProps[]>>;
```
?